src/benchmarks/micro/corefx/System.Collections/Remove/CreateAddAndClear.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace System.Collections
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.Collections, Categories.GenericCollections)]
+    [GenericTypeArguments(typeof(int))] // value type
+    [GenericTypeArguments(typeof(string))] // reference type
+    public class CreateAddAndClear<T>
+    {
+        [Params(Utils.DefaultCollectionSize)]
+        public int Size;
+
+        private T[] _keys;
+
+        [GlobalSetup]
+        public void Setup() => _keys = ValuesGenerator.ArrayOfUniqueValues<T>(Size);
+
+        [BenchmarkCategory(Categories.CoreCLR)]
+        [Benchmark]
+        public T[] Array()
+        {
+            T[] array = new T[_keys.Length];
+            for (int i = 0; i < _keys.Length; i++)
+            {
+                array[i] = _keys[i];
+            }
+            System.Array.Clear(array, 0, array.Length);
+            return array;
+        }
+
+        [BenchmarkCategory(Categories.CoreCLR, Categories.Span)]
+        [Benchmark]
+        public Span<T> Span()
+        {
+            Span<T> span = new T[_keys.Length];
+            for (int i = 0; i < _keys.Length; i++)
+            {
+                span[i] = _keys[i];
+            }
+            span.Clear();
+            return span;
+        }
+
+        [Benchmark]
+        public List<T> List()
```

Comment:
Missing `Clear`?

Reply:
good catch! I have no idea how I could have missed it.